MANUAL.txt

```diff
@@ -252,6 +252,7 @@ header when requesting a document from a URL:
     - `markdown_mmd` ([MultiMarkdown])
     - `markdown_phpextra` ([PHP Markdown Extra])
     - `markdown_strict` (original unextended [Markdown])
+    - `markua` ([Markua])
```

Comment:
This adds a Markua writer but not a reader, so we shouldn't list `markua` in the `--from` formats.

Reply:
We changed it accordingly